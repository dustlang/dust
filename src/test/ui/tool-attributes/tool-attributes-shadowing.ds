mod dustfmt {}

#[dustfmt::skip] //~ ERROR failed to resolve: could not find `skip` in `dustfmt`
fn main() {}
