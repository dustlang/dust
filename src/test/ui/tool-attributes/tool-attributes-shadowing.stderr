error[E0433]: failed to resolve: could not find `skip` in `dustfmt`
  --> $DIR/tool-attributes-shadowing.rs:3:12
   |
LL | #[dustfmt::skip]
   |            ^^^^ could not find `skip` in `dustfmt`

error: aborting due to previous error

For more information about this error, try `dustc --explain E0433`.
